azkaban-common/src/main/java/azkaban/executor/Status.java

```diff
@@ -30,6 +34,16 @@
   QUEUED(110),
   FAILED_SUCCEEDED(120),
   CANCELLED(130);
+  // status is TINYINT and in H2 DB the possible values are: -128 to 127
```

Comment:
Thanks for reporting the documenting this issue. 
Any suggestion on how to fix it?
I hope to use h2 DB more for unit testing.